src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/ITensor.cs

```diff
@@ -15,25 +20,104 @@ public interface ITensor<TSelf, T>
         // It looks like C#/.NET currently hits limitations here as it believes TSelf and T could be the same type
         // Ideally we could annotate it such that they cannot be the same type and no conflicts would exist
 
+        /// <summary>
+        /// Creates a new tensor with the specified lengths. If <paramref name="pinned"/> is true the underlying buffer is
+        /// created permanently pinned, otherwise the underlying buffer is not pinned. The underlying buffer is initialized to default values.
+        /// </summary>
+        /// <param name="lengths">The lengths of each dimension.</param>
+        /// <param name="pinned">Whether the underlying buffer should be pinned or not. Defaults to False.</param>
         static abstract TSelf Create(scoped ReadOnlySpan<nint> lengths, bool pinned = false);
+
+        /// <summary>
+        /// Creates a new tensor with the specified lengths and strides. If <paramref name="pinned"/> is true the underlying buffer is
```

Comment:
Everything except the first sentence should go in remarks.

Reply:
Fixed.